apps/backoffice-v2/src/pages/Home/Home.page.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FunctionComponent } from 'react';
+import { NavLink, Outlet } from 'react-router-dom';
+import { UserAvatar } from '@/common/components/atoms/UserAvatar/UserAvatar';
+import { Tabs } from '@/common/components/organisms/Tabs/Tabs';
+import { TabsList } from '@/common/components/organisms/Tabs/Tabs.List';
+import { TabsTrigger } from '@/common/components/organisms/Tabs/Tabs.Trigger';
+import { TabsContent } from '@/common/components/organisms/Tabs/Tabs.Content';
+import { DateRangePicker } from '@/common/components/molecules/DateRangePicker/DateRangePicker';
+import { useHomeLogic } from '@/common/hooks/useHomeLogic/useHomeLogic';
+
+export const Home: FunctionComponent = () => {
+  const {
+    handleDateRangeChange,
+    from,
+    to,
+    t,
+    firstName,
+    fullName,
+    avatarUrl,
+    statisticsLink,
+    workflowsLink,
+    pathname,
+    locale,
+  } = useHomeLogic();
+
+  return (
+    <div className={`flex flex-col gap-10 p-10`}>
+      <div className={`flex items-center justify-between`}>
+        <div className={`flex items-center`}>
+          <UserAvatar
+            fullName={fullName ?? ''}
+            className={`mr-2 d-6`}
+            avatarUrl={avatarUrl ?? undefined}
+          />
+          <h3 className={`flex max-w-[45ch] break-all text-2xl font-semibold`}>
+            {t(`home.greeting`)}
+            {firstName && ` ${firstName}`}
+          </h3>
+        </div>
+        <DateRangePicker
+          onChange={handleDateRangeChange}
+          value={{ from: from ? new Date(from) : undefined, to: to ? new Date(to) : undefined }}
+        />
+      </div>
+      <div>
+        <Tabs defaultValue={pathname} key={pathname}>
+          <TabsList>
+            <TabsTrigger asChild={true} value={`/${locale}/home/statistics`}>
```

Comment:
Is the active tab state affected by the search query param change? Do we need to also pass `search` into `value`?

Reply:
Yes, the active state is affected by the search query parameter change. This is the reason I used the '/${locale}/home/statistics' and '/${locale}/home/workflows' instead of 'statisticsLink' and 'workflowsLink' directly into value.